ods_ci/tests/Tests/100__deploy/100__installation/112__model_registry.robot

```diff
@@ -0,0 +1,56 @@
+*** Settings ***
+Documentation       Post install test cases that verify Model Registry
+Library             OpenShiftLibrary
+Resource            ../../../Resources/Page/ODH/JupyterHub/HighAvailability.robot
+Resource            ../../../Resources/OCP.resource
+Suite Teardown      Teardown
+
+
+*** Variables ***
+${MODEL_REGISTRY_NS}=    ${APPLICATIONS_NAMESPACE}
+${ENABLED_REGISTRY}=     False
+
+
+*** Test Cases ***
+Verify Model Registry Operator Installation
+    [Documentation]    Verifies that the Model Registry operator has been
+    ...    deployed in the ${APPLICATIONS_NAMESPACE} namespace in ODS
+    [Tags]    OpenDataHub    robot:recursive-continue-on-failure    RunThisTest
+    ${modelregistry} =    Is Component Enabled    modelregistry    ${DSC_NAME}
+    IF    "${modelregistry}" == "false"
+        Enable Component    modelregistry
+        ${ENABLED_REGISTRY} =  Set Variable  True
+    END
```

Comment:
you can use `Set Component State` keyword. it uses "oc apply", hence if it's enabled already it leaves it as it is

Reply:
this comment goes in hand with https://github.com/red-hat-data-services/ods-ci/pull/1295#discussion_r1576046658